lib/flame/controller/actions.rb

```diff
@@ -33,21 +33,37 @@ def inherit_actions(actions = superclass.actions, exclude: [])
 			end
 
 			## Re-define public instance method from module
+			## @param exclude [Array<Symbol>] Actions for excluding
+			##   from module public instance methods
+			## @param only [Array<Symbol>, nil] Actions for re-defining
+			##   from module public instance methods
```

Comment:
`mod` param too, please.

Reply:
changed

Reply:
Are you sure?

Reply:
i do not understand what you are wanting

Reply:
`with_actions` has the first `mod` argument. YARD for this method doesn't have it's description.

Reply:
added